_posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md

```diff
@@ -0,0 +1,89 @@
+---
+title: New API Blueprint tutorial
+excerpt: Learning API Blueprint has never been easier.
+layout: post 
+date: 2015-03-03 16:00:00 +0000
+author: zdenek 
+published: true
+comments: true
+---
+
+Superb and concise documentation is one of our priorities at Apiary. But frankly, we weren't doing the most stellar job when it came to documenting our very own product and the plethora of awesome tools we are working on. 
+
+This is changing right now. Allow me to introduce you to our brand new and much simplified API Blueprint tutorial. 
+
+We have completely redesigned the tutorial from the ground. Stripped it from all clutter. Saving advanced topics for the later.  The new tutorial no longer expects any knowledge of Markdown. You can get started with API Blueprint in less than thousands words counting blueprint snippets and footnotes! 
```

Comment:
ground up, stripped any clutter, and saved advanced topics for later. 

BTW, not sure what you mean by that last statement. If it means included them at the end, that phraseology does not really work.